src/main/java/com/backbase/ct/bbfuel/client/approval/ApprovalIntegrationRestClient.java

```diff
@@ -36,7 +36,7 @@
     private final BbFuelConfiguration config;
 
     private static final String SERVICE_VERSION = "v2";
-    private static final String CLIENT_API = "client-api";
+    private static final String CLIENT_API = "integration-api";
```

Comment:
Can you change the variable name to 'INTEGRATION-API'? it makes a bit more sense like that :D

Reply:
done